features/other_b/series_lock.feature

```diff
@@ -65,3 +65,22 @@ Feature: Locked and partially locked series
       And I should see "Part 1 of Antiholidays"
     When I press "Update"
     Then I should see "Work was successfully updated"
+
+  Scenario: Admins can see locked series on user's series page
+    Given I am logged in as "Accumulator"
+      And I post the work "Restricted work" as part of a series "So Restrictive"
+      And I lock the work "Restricted work"
```

Comment:
If we want to avoid the double save, we could do this to check the restricted box before posting:

```
I set up the draft etc etc
I lock the work
I post the work
```

But the phrasing on your version is nicer, so I don't mind either way!